README.rst

```diff
@@ -1,19 +1,111 @@
+####################
 edx-name-affirmation
+####################
+
+| |Build Status| |Coveralls|
+
+This library contains data model and business logic to store verified 
+personal information for learners of the edx-platform.
+This library is designed to be a pluggable library to the edx-platform.
+
+Overview
+========
+
+This library mainly stores learner's various stages of a learner's verified identifying information.
+The prevalent example is the learner's full name. To verify the full name, the learner
+has to first initate from UI that is part of the platform.
+Here are the steps of which the PII information, for example: full name, goes through:
+
+1. learner requests to update the name
```

Comment:
I think we should mention that this is done through the account MFE. I don't think the legacy account experience supports name-affirmation.

Reply:
On that note, can we list out what needs to be running for this to work out of the box? `frontend-app-account` is a big one. I don't believe this works with legacy account or IDV flows.